### ISSUE TITLE
How to query single Type content ? 

### ISSUE THREAD

**Issue by juliensl** [strapi/gatsby-source-strapi#119]
Hi everybody ! 

I am trying to use this plugin to pull some data from my Strapi.
I succeed to pull data from a contentTypes : 

```
      resolve: "gatsby-source-strapi",
      options: {
        apiURL: "http://....:1337",
        contentTypes: [
          // List of the Content Types you want to be able to request from Gatsby.
          "article",
        ],
        queryLimit: 1000,
      },
```
```
query {
          allStrapiArticle {
            edges {
              node {
                id
                title
                test2
              }
            }
          }
        }
```

But now, I would like to query content from a Single Type content, here "accueil" : 
```
      resolve: "gatsby-source-strapi",
      options: {
        apiURL: "http://.....:1337",
        contentTypes: [
          // List of the Content Types you want to be able to request from Gatsby.
          "article",
        ],
        singleTypes: ["accueil"],
        queryLimit: 1000,
      },
```

I tried to write my query asking `allStrapiAccueil`, `strapiAccueil`, `accueil`, but it does not work.
```
  query CampusQuery {
    allStrapiAccueil {
      edges {
        node {
          id
        }
      }
    }
  }
```

Which syntax should I use ? 

I think that it could be nice to write the good syntax in the readme ;) 
( I am tagging a colleague who is working on it too @knabeth )

Thank you :)

**Comment by mannp** [comment]
Your bottom query is the same as mine.

Does your singleTypes definition match your API ID : <accueil> ? for that page in Strapi?

**Comment by juliensl** [comment]
Hi @mannp , thank you for your answer.

Yep, the ID match : 
![api id](https://user-images.githubusercontent.com/43228020/78939998-6d524980-7ab5-11ea-9159-532c2730f765.png)

**Comment by juliensl** [comment]
Ok, solved just here : #116 
I was using the package version number 0.0.11.... :cry: 

It works now, thank you manp ;)
